corehq/apps/hqwebapp/urls.py

```diff
@@ -52,6 +56,38 @@
     url(r'^login/$', domain_login, name='domain_login'),
     url(r'^login/mobile/$', domain_login, name='domain_mobile_login',
         kwargs={'template_name': 'login_and_password/mobile_login.html'}),
+
+    # This url is linked to from web login page
+    url(r'^reset_pwd_email/$', exception_safe_password_reset,
+        {'template_name': 'login_and_password/password_reset_form.html',
+         'email_template_name': 'login_and_password/domain_reset_pwd_email.html',
+         'from_email': settings.DEFAULT_FROM_EMAIL,
+         'extra_context': {'current_page': {'page_name': _('Password Reset')}}},
+        name='domain_reset_pwd_email'),
+    # This url is linked to from mobile login page
```

Comment:
is the mobile login page ever called? It looks like only when this is true:
```py
def is_mobile_url(url):
    # Minor hack
    return ('reports/custom/mobile' in url)
```
and I don't think that is ever true any more?